src/CanMute.php

```diff
@@ -4,13 +4,14 @@
 
 trait CanMute
 {
-    public static function mute($events = null)
+    /** @param string|string[] $events */
```

Comment:
@Sergej-Tihonov Is this redundant, given the type hinting in the function signature?

Reply:
Fair point.